exec_test.go

```diff
@@ -64,32 +65,32 @@ func TestCommandRun(t *testing.T) {
 	)
 
 	testCases := map[string]struct {
-		cmd     string
-		timeout time.Duration
-
+		cmd        string
+		timeout    time.Duration
 		fakeDistro bool
 		cancelOn   when
 
-		wantErr       bool
+		errWanted     bool
```

Comment:
honest question: We generally use `want` when we get a single output that we want, then, we can have `wantFirst`, `wantSecond` or `wantFoo` depending on what we are asserting against.

Following, this pattern, I see in general `wantErr` and this is what we use accross project. Do you think that semantically, this is wrong? (I like that all "wants" are prefixed the same way, and you go simply to the "want" section.